### PR TITLE
10/11 blog

### PR DESCRIPTION
**Pull request by cnunciato**
URL path: `/blog/developer-portal-platform-teams/`

### CODE REVIEW

File: themes/default/content/blog/developer-portal-platform-teams/index.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+* 🆕 Pulumi Backstage Plugin:  For teams using Backstage, they can now integrate Pulumi directly within their Backstage instance, via support for a new Pulumi plugin as well as new Pulumi scaffolder actions.
+* 🆕 `pulumi new` support for private templates: By default, `pulumi new` provides a number of templates provided by Pulumi, but it can also use your own [private templates](/docs/cli/commands/pulumi_new/).
+
+__Policy and Compliance__: Platform teams need robust and flexible solutions for establishing policy and compliance guardrails for their cloud infrastructure. Pulumi’s existing CrossGuard policy-as-code framework provides the richest option for defining and enforcing policy across all infrastructure in an organization, using the full capabilities of your favorite languages.  Today, we are extending that support with two important new features for platform teams:
```
Comment (toriancrane) [inline themes/default/content/blog/developer-portal-platform-teams/index.md:31]: Should we link to a relevant CrossGuard PaS doc here?

File: themes/default/content/blog/developer-portal-platform-teams/index.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+tags:
+    - developer-portals
+    - templates
+    - policy-as-code
+    - backstage
+---
+
+Over the last two years, we’ve seen a huge surge in adoption of Pulumi by __Platform Teams__ -- centralized teams within a business responsible for building out core cloud infrastructure and providing tools to the rest of the organization to maximize the productivity, cost efficacy, compliance and velocity of application and service delivery throughout the organization. These teams use Pulumi to manage their own cloud infrastructure complexity, to offer best practices components to their organizations, to enforce organizational policy, and to drive infrastructure delivery automation.
+
+Today, we’re excited to announce a broad range of new features that extend Pulumi Cloud’s support for Platform Teams, across the entire lifecycle of cloud delivery.
```
Comment (interurban) [inline themes/default/content/blog/developer-portal-platform-teams/index.md:23]:
```suggestion
Today, we’re excited to announce a broad range of new features that extend (Pulumi Cloud’s support for Platform Teams)[/product/internal-developer-platforms/], across the entire lifecycle of cloud delivery.
```

Comment (interurban) [reply themes/default/content/blog/developer-portal-platform-teams/index.md:23]: adding a link to the product page.

**Review by interurban — APPROVED** [review]
@cnunciato can you add a link back to the product page before we ship.

**Pull request merged**